### PR TITLE
Bflix Vidstream server fix

### PR DESCRIPTION
**Pull request by Stormunblessed**

### CODE REVIEW

**Comment by Stormunblessed** [comment]
Sometimes it will give 2004 error, but if you go back and select it again it'll work, weird thing

File: app/src/main/java/com/lagradost/cloudstream3/movieproviders/BflixProvider.kt [hunk 1]
```diff
@@ -317,10 +317,12 @@ class BflixProvider(providerUrl: String, providerName: String) : MainAPI() {
         )
             .select("html body #episodes").map {
                 val tvType = if (data.contains("movie/")) TvType.Movie else TvType.TvSeries
-                val cleandata = data.replace(mainUrl,"").replace("/1-full","")
+                val cleandata = data.replace(mainUrl,"")
                 val servers = if (tvType == TvType.Movie) it.select(".episode a").attr("data-ep")
                 else
                     it.select(".episode a[href=$cleandata]").attr("data-ep")
+                        ?: it.select(".episode a[href=${cleandata.replace("/1-full","")}]").attr("data-ep")
+                println(servers)
```
Comment (LagradOst) [inline app/src/main/java/com/lagradost/cloudstream3/movieproviders/BflixProvider.kt:325]: remove println

**Comment by LagradOst** [comment]
Ready to merge?

**Comment by Stormunblessed** [comment]
> Ready to merge?

Yeah

**Pull request merged**